### ISSUE TITLE
All scripts break when disconnected from server.

### ISSUE THREAD

**Issue by rpgwaiter** [ORelio/Minecraft-Console-Client#204]
I'm running MCC on linux through Mono (not sure if that's relevant), and any time that one of the clients gets disconnected from the server, it does this weird thing where ">242;3R" is at the front of the write line and you can't backspace it away. You have to type /reco quite a few times in order for it to reconnect. Auto-reconnect doesn't work, nor does a script that should run when I get kicked from server. Here's a video demonstrating it. https://youtu.be/ChVkXHg0VJ0

**Comment by ORelio** [comment]
Hello,

`242;3R` is a terminal control code, usually Mono will interact with your terminal using control sequences inserted into what you usually type using your keyboard (STDIN). The terminal detects control codes and acts accordingly eg by moving the cursor or changing colors. If control codes expects a response eg current cursor coordinates (which MCC needs for handling carriage returns), the terminal writes back a response in what is usually displayed to screen (STDOUT). The handling app will usually detect responses and act accordingly.

Simply put: Depending on your terminal/mono combination, some control codes may glitch out, that should not be troublesome, the only (harmless) effect is that you get some control codes printed on screen.

Regarding reconnection: Are you sure you enabled and configured the AutoRelog bot? All features are turned off by default, see README and MinecraftClient.ini files. You need to enabled AutoRelog and create a file with keywords for triggering auto relogs based on text you got as kick message.

**Comment by ORelio** [comment]
The initial problem should have been answered, feel free to reopen if needed :)
